cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -0,0 +1,371 @@
+name "Google Committed Use Discount Recommender"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy finds Google Committed Use Discount Recommendations and reports when it finds them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/google/cud_recommendations) and [docs.rightscale.com/policies](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+category "Cost"
+severity "low"
+default_frequency "daily"
+info(
+  version: "2.0",
```

Comment:
If requires changelog updates then don't forget to update the template version number here